scripts/ingests/ingest_ATLAS_spectral_types.py

```diff
@@ -1,38 +1,135 @@
-#------------------------------------------------------------------------------------------------
+# ------------------------------------------------------------------------------------------------
 
 from astrodbkit2.astrodb import create_database
 from astrodbkit2.astrodb import Database
+from simple.schema import *
 from astropy.table import Table
 import numpy as np
 import re
+import os
 from utils import convert_spt_string_to_code
 
-connection_string = 'sqlite:///../../SIMPLE.db'  # SQLite
-create_database(connection_string)
-db = Database(connection_string)
-db.load_database('../../data')
+DRY_RUN = True
+RECREATE_DB = False
+VERBOSE = False
+
+verboseprint = print if VERBOSE else lambda *a, **k: None
+
+db_file = 'SIMPLE.db'
+db_connection_string = 'sqlite:///SIMPLE.db'  # SQLite
+
+try:
+	db_file_path = db_file.resolve(strict=True)
+except:
+	# SIMPLE.db file does not exist so create it
+	create_database(db_connection_string)
+	db = Database(db_connection_string)
+	db.load_database('data')
+else:
+	# SIMPLE.db file does exist
+	if RECREATE_DB: # Recreate database anyway
+		os.remove(db_file)
```

Comment:
Why not use `if os.path.exists(db_file): os.remove(db_file)` to check if the file exists and remove it rather than the whole try/except/else?

Reply:
the try/except/else handles the situation where `RECREATE_DB = False` but Simple.db file does not exist.  I agree that this could be written with `if`s, but the try/except/else feels cleaner to me.